### PR TITLE
Issue #97 - support SDE/EGDB

### PR DESCRIPTION
**Pull request by csmoore**
Add support for SDE - see #97

### CODE REVIEW

**Comment by csmoore** [comment]
@kgonzago or @topowright - can one of you review/merge - thanks

**Comment by csmoore** [comment]
Just checking back to see if this can be reviewed/approved, I have another change to one of these files that will create a conflict if I don't merge this first.

File: source/ProSymbolEditor/Models/MilitaryOverlayDataModel.cs [hunk 1]
```diff
@@ -33,60 +33,84 @@ public class MilitaryOverlayDataModel
...
-                featureClassExists.Add("LandEquipment", false);
-                featureClassExists.Add("METOCAreas", false);
-                featureClassExists.Add("METOCLines", false);
-                featureClassExists.Add("METOCPoints", false);
-                featureClassExists.Add("SeaSubsurface", false);
-                featureClassExists.Add("SeaSurface", false);
-                featureClassExists.Add("SIGINT", false);
-                featureClassExists.Add("Space", false);
-                featureClassExists.Add("Units", false);
+                featureClassExists.Add(prefixName + "Activities", false);
```
Comment (kgonzago) [inline source/ProSymbolEditor/Models/MilitaryOverlayDataModel.cs]: @csmoore Can "Activities", "Air", etc. be encapsulated in an enum?

Comment (csmoore) [inline source/ProSymbolEditor/Models/MilitaryOverlayDataModel.cs]: I didn't really think about, but this class could indeed use the existing list of these from SymbolSetMappings class - the project would just need refactored some to move the object hierarchy/containers a bit - just let me know if you want me to do as part of this PR

**Pull request merged**